keyboards/blank_tehnologii/manibus/keymaps/via/keymap.c

```diff
@@ -0,0 +1,103 @@
+       _______, _______, _______, _______, _______, _______,                         _______, _______, _______, _______, _______, _______,
+//   ├────────┼────────┼────────┼────────┼────────┼────────┤                       ├────────┼────────┼────────┼────────┼────────┼────────┤
+       _______, _______, _______, _______, _______, _______,                         _______, _______, _______, _______, _______, _______,
+//   ├────────┼────────┼────────┼────────┼────────┼────────┤                       ├────────┼────────┼────────┼────────┼────────┼────────┤
+       _______, _______, _______, _______, _______, _______,                         _______, _______, _______, _______, _______, _______,
+//   ├────────┼────────┼────────┼────────┼────────┼────────┤                       ├────────┼────────┼────────┼────────┼────────┼────────┤
+       _______, _______, _______, _______, _______, _______,                         _______, _______, _______, _______, _______, _______,
+//   ├────────┼────────┼────────┼────────┼────────┼────────┼────────┐     ┌────────┼────────┼────────┼────────┼────────┼────────┼────────┤
+       _______, _______, _______, _______, _______, _______, _______,       _______, _______, _______, _______, _______, _______, _______
+//   └────────┴────────┴────────┴────────┴────────┴────────┴────────┘     └────────┴────────┴────────┴────────┴────────┴────────┴────────┘
+  ),
+  [_FN3] = LAYOUT(
+//   ┌────────┬────────┬────────┬────────┬────────┬────────┐                       ┌────────┬────────┬────────┬────────┬────────┬────────┐
+      _______, _______, _______, _______, _______, _______,                         _______, _______, _______, _______, _______, _______,
+//   ├────────┼────────┼────────┼────────┼────────┼────────┤                       ├────────┼────────┼────────┼────────┼────────┼────────┤
+      _______, _______, _______, _______, _______, _______,                         _______, _______, _______, _______, _______, _______,
+//   ├────────┼────────┼────────┼────────┼────────┼────────┤                       ├────────┼────────┼────────┼────────┼────────┼────────┤
+      _______, _______, _______, _______, _______, _______,                         _______, _______, _______, _______, _______, _______,
+//   ├────────┼────────┼────────┼────────┼────────┼────────┤                       ├────────┼────────┼────────┼────────┼────────┼────────┤
+      _______, _______, _______, _______, _______, _______,                         _______, _______, _______, _______, _______, _______,
+//   ├────────┼────────┼────────┼────────┼────────┼────────┼────────┐     ┌────────┼────────┼────────┼────────┼────────┼────────┼────────┤
+      _______, _______, _______, _______, _______, _______, _______,       _______, _______, _______, _______, _______, _______, _______
+//   └────────┴────────┴────────┴────────┴────────┴────────┴────────┘     └────────┴────────┴────────┴────────┴────────┴────────┴────────┘
+  )
```

Comment:
This still needs removing, otherwise it will not compile
```suggestion
  )
```